dbtmetabase/metabase.py

```diff
@@ -819,6 +823,10 @@ def _build_exposure(
                 if exposure.upper() in refable_models
             ],
         }
+        if creator_email is None:
+            del exposure["owner"]
```

Comment:
Deleting the key if the variable is not defined works around a type error that came up when doing the more obvious approach of setting the key if the variable is defined. There could be a more elegant solution here...